Sources/JWTKit/JWTSigners.swift

```diff
@@ -229,6 +229,21 @@ private struct JWKSigner {
             default:
                 return nil
             }
+
+        case .octetKeyPair:
+            guard let algorithm = algorithm ?? self.jwk.algorithm else {
+                return nil
+            }
+                
+            switch algorithm {
+                case .eddsa:
+                    guard let key = try? EdDSAKey(x: self.jwk.x, d: self.jwk.privateExponent, curve: self.jwk.curve ?? .ed25519) else {
```

Comment:
Is defaulting this to ed25519 right?

Reply:
Yes, it's the only curve supported for `EdDSA` right now. I made it more explicit with https://github.com/vapor/jwt-kit/pull/82#discussion_r993942834

Reply:
I removed the default fallback. Any unknown curve will not create a signer anymore. 
https://github.com/vapor/jwt-kit/pull/82/commits/12f9a890c5a5829b10c1ff262c128288e745e5cb